### DISCUSSION TITLE
I am particularly looking forward to the update of Gemma2.

### DISCUSSION THREAD

**Discussion by Zhyblx** [amithkoujalgi/ollama4j#53] [Q&A] [answered]
Hello
 
Can new models such as qwen2 and Gemma2 be updated into the project as soon as possible? I am particularly looking forward to the update of Gemma2.
 
Thank！

**Comment by amithkoujalgi** [comment, marked as answer]
Thanks for request. `qwen2` and `gemma2` have been added to library and you should be able to use that in the latest version of Ollama4j. 

Just FYI, you don't need to wait for library updates if you just want to use newer models available in Ollama. You could still pass the model name as a `String` as all the following methods use the argument `String model` as an argument:

- `pull()`
- `getModelDetails()`
- `chat()`
- `generate()`
- `generateWithImageURLs()`
- `generateWithImageFiles()`
- `delete()`
- `generateEmbeddings()`

**Reply by Zhyblx** [reply]
thank you very much！
Because your project has made me feel that the world is so beautiful!

**Comment by Zhyblx** [comment]
"Thank you, I have cloned your project a few days ago and carefully read your code. I have understood how to use the latest model. Also, thank you again for providing the project."
发自我的 iPhone

在 2024年7月12日，02:13，Amith Koujalgi ***@***.***> 写道：


Thanks for request. qwen2 and gemma2 have been added to library and you should be able to use that in the latest version of Ollama4j.

Just FYI, you don't need to wait for library updates if you just want to use newer models available in Ollama. You could still pass the model name as a String as all the following methods use the argument String model as an argument:

  *   pull()
  *   getModelDetails()
  *   chat()
  *   generate()
  *   `generateWithImageURLs()
  *   generateWithImageFiles()
  *   delete()
  *   generateEmbeddings()

**Comment by Zhyblx** [comment]
I Appreciate it really

发自我的 iPhone

在 2024年7月12日，22:57，张 益斌 ***@***.***> 写道：

﻿ "Thank you, I have cloned your project a few days ago and carefully read your code. I have understood how to use the latest model. Also, thank you again for providing the project."
发自我的 iPhone

在 2024年7月12日，02:13，Amith Koujalgi ***@***.***> 写道：


Thanks for request. qwen2 and gemma2 have been added to library and you should be able to use that in the latest version of Ollama4j.

Just FYI, you don't need to wait for library updates if you just want to use newer models available in Ollama. You could still pass the model name as a String as all the following methods use the argument String model as an argument:

  *   pull()
  *   getModelDetails()
  *   chat()
  *   generate()
  *   `generateWithImageURLs()
  *   generateWithImageFiles()
  *   delete()
  *   generateEmbeddings()

**Comment by amithkoujalgi** [comment]
Thanks! If your question has been answered, please consider clicking on the “Mark as answer” button for my [response](https://github.com/amithkoujalgi/ollama4j/discussions/53#discussioncomment-10024112).

**Comment by Zhyblx** [comment]
Sorry for my slow response. As a native Chinese speaker, English is a bit awkward for me. But I should have already marked it on the platform. If there are any mistakes made, please let me know by email.

发件人: Amith Koujalgi ***@***.***>
日期: 星期日, 2024年7月14日 03:46
收件人: amithkoujalgi/ollama4j ***@***.***>
抄送: Zhyblx ***@***.***>, Author ***@***.***>
主题: Re: [amithkoujalgi/ollama4j] I am particularly looking forward to the update of Gemma2. (Discussion #53)

Thanks! If your question has been answered, please consider clicking on the “Mark as answer” button for my response.

―
